### ISSUE TITLE
CS:GO Crash

### ISSUE THREAD

**Issue by LuvFhu** [KittenPopo/csghost-faq#18]
I can launch Ghost and load into the main menu, load config and everything, but the second I load into the actual match I crash, I made sure all my av's were off and I tried multiple times, and it still crashes.

**Comment by zedtao** [comment]
Yeah the update on April 25th. Most injectors have stopped working.
